std-bits/table/src/main/java/org/enso/table/data/column/storage/LongStorage.java

```diff
@@ -291,6 +236,18 @@ public long doLong(long in, long arg) {
                 return in % arg;
               }
             })
+        .add(
+            new LongNumericOp(Maps.POWER, true) {
+              @Override
+              public double doDouble(long in, double arg) {
+                return Math.pow(in, arg);
+              }
+
+              @Override
+              public long doLong(long in, long arg) {
+                throw new UnsupportedOperationException("Power operation should cast to double.");
```

Comment:
```suggestion
                throw new IllegalStateException("Internal error: Power operation should cast to double.");
```

nitpick, but I think this can be illegal state since by design this branch should be unreachable. It's not that this operation is not supported, it's that if we somehow reached this branch we have a bug in our code.